app/command_test.go

```diff
@@ -75,6 +75,45 @@ func TestCreateCommandPost(t *testing.T) {
 	}
 }
 
+func TestExecuteCommand(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	// Testing slash command message is correctly parsed
+	TestCases := []string{
+		"/code happy path",
+		"/code\nnewline path",
+		"/code\n/nDouble newline path",
+		"/code  double space",
+		"/code\ttab",
+	}
+	results := []string{
+		"    happy path",
+		"    newline path",
+		"    /nDouble newline path",
+		"     double space",
+		"    tab",
+	}
+
+	for index, TestCase := range TestCases {
+		args := &model.CommandArgs{
+			Command: TestCase,
+			T:       func(s string, args ...interface{}) string { return s },
+		}
+		resp, _ := th.App.ExecuteCommand(args)
+
+		assert.Equal(t, resp.Text, results[index])
+	}
+	argsMissingSlashCharacter := &model.CommandArgs{
```

Comment:
suggest wrapping this in a subtest:
```go
t.Run("missing slash character", func(t *testing.T) {
    	argsMissingSlashCharacter := &model.CommandArgs{
                // ...
        }
        // ...
})
```

and then the above block in a subtest named `valid commands` or some such.

Reply:
Given my `strings.HasPrefix` comment above, can we also add one more test that asserts the behaviour of an empty `args.Command` string?

Reply:
absolutely